src/nginx/t/backend_routing_constant_address.t

```diff
@@ -203,4 +243,42 @@ EOF
   $server->run();
 }
 
+
+sub metadata {
+  my ($t, $port, $file) = @_;
+  my $server = HttpServer->new($port, $t->testdir() . '/' . $file)
+    or die "Can't create test server socket: $!\n";
+  local $SIG{PIPE} = 'IGNORE';
+
+  $server->on('GET', '/computeMetadata/v1/instance/service-accounts/default/token', <<'EOF');
+HTTP/1.1 200 OK
+Metadata-Flavor: Google
+Content-Type: application/json
+
+{
+ "access_token":"ya29.7gFRTEGmovWacYDnQIpC9X9Qp8cH0sgQyWVrZaB1Eg1WoAhQMSG4L2rtaHk1",
+ "expires_in":200,
+ "token_type":"Bearer"
+}
+EOF
+
+  $server->on('GET', '/computeMetadata/v1/instance/service-accounts/default/identity?audience=test-audience',  <<"EOF");
+HTTP/1.1 200 OK
+Metadata-Flavor: Google
+Content-Type: application/json
+
+test_audience_override\r\n\r\n
```

Comment:
Should the JSON format be the same as default token?  same as line 258-262?

Reply:
No, this is JWT token,  not in access token format.

https://cloud.google.com/compute/docs/instances/verifying-instance-identity#token_format

The only option is FORMAT=full | standard, but both returns a string.

And also, there is no expiration time returned.  I set the kInstanceIdentityTokenExpiration = 3500, same as that of kClientSecretAuthTokenExpiration.  Is there any concern about this expiration time?

Reply:
We may have to parse the JWT to find out the expiration time.

Reply:
Yes, the expiration time is inside the JWT. It is almost certainly going to remain 1 hour, but we do not actually have a guarantee of that.

Reply:
for default token, I know that it is not fresh new,  it could expire any where in between 1m to 60m.
Could you use jwk.io to open the JWT token from identidy, to see if it is always a fresh new one with 55m minutes more to go.   or it just like default token.

If the former, we are OK, we just use a hard coded expiration for now.
If the latter, we have to add TODO, and use JWT parser code to get expiration

Reply:
Add TODO.